README.md

```diff
@@ -38,6 +38,7 @@ A Terminal Emulator based on UWP and web technologies.
 
 ### Chocolatey package manager installation
 
+- In **Settings**, click **For Developers**, and select **Developer mode**.
```

Comment:
Do we need to call out the necessary reboot, or are we content with letting Windows bug them?

Reply:
I think Windows will be reboot encouraging enough

Reply:
Weirdly I didn't need to reboot to switch from Sideload to Developer Mode.